packages/editor/src/plugins/fill-in-the-blanks-exercise/components/blank-controls.tsx

```diff
@@ -87,6 +92,11 @@ export function BlankControls(props: BlankControlsProps) {
     }px`
   }, [editor, selectedElement])
 
+  const isBlankAnswerAlphabetical = useMemo(() => {
+    if (correctAnswers[0].length === 0) return true
+    return /^[a-zA-Z]+$/.test(correctAnswers[0])
+  }, [correctAnswers])
```

Comment:
Still not 💯% sure about that one since it shows basically a math-only feature on all blanks with e.g. punctuation marks. But it's not a blocker at this point since we now fixed the evaluation code.

Reply:
As soon as there's clarity on when exactly to show it, I'll be happy to adjust the code 😁